api/app/controllers/transcript.controller.ts

```diff
@@ -366,12 +370,28 @@ export async function claim(req: Request, res: Response) {
     return;
   }
 
+  // if user has successfully reviewed fewer than 3 transcripts
+  // allow to claim only 1 transcript and return if user has already has a pending review
+  // if user has successfully reviewed 3 or more transcripts, allow to have 6 pending reviews
+  const successfulReviews = await Review.findAll({
```

Comment:
nit: let's call this `mergedReviews` to start be more consistent with naming of variables